### PR TITLE
Updated launch file to use ros_ign_gazebo

### PR DESCRIPTION
**Pull request by ahcorde**
This PR builds on top of this other PR #80

This PR update the lanch file to use the launch file inside ros_ign_gazebo

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

File: ros_ign_gazebo_demos/launch/air_pressure.launch.py
Comment (chapulina) [inline ros_ign_gazebo_demos/launch/air_pressure.launch.py]: This doesn't work yet, right? See https://github.com/ignitionrobotics/ros_ign/issues/78

File: ros_ign_gazebo_demos/launch/battery.launch.py
Comment (chapulina) [inline ros_ign_gazebo_demos/launch/battery.launch.py:36]: Is there a reason not to keep this in a variable? It's maybe a bit cleaner?

**Review by chapulina — COMMENTED** [review]
Can we rebase this on top of #80 ? Otherwise it's hard to test if things are working.

Also note that `ros_ign_gazebo_demos/README.md` will need updates too.

**Comment by chapulina** [comment]
#80 and #81 have been merged. I think this needs to be rebased and maybe needs a couple of tweaks.

File: ros_ign_gazebo_demos/launch/image_bridge.launch.py
Comment (chapulina) [inline ros_ign_gazebo_demos/launch/image_bridge.launch.py:52]: This is only [changing from Foxy](https://index.ros.org/doc/ros2/Releases/Release-Foxy-Fitzroy/#launch-ros), so we can't land it on the `dashing` branch.

**Review by chapulina — CHANGES_REQUESTED** [review]
Before merging, we should make sure that all `ros_ign_gazebo_demos` work with dashing on this branch.

**Comment by chapulina** [comment]
See #85 for a Dashing-compatible version.

**Comment by ahcorde** [comment]
Related with this issue https://github.com/ignitionrobotics/ros_ign/pull/89

File: ros_ign_gazebo_demos/launch/air_pressure.launch.py
Comment (chapulina) [inline ros_ign_gazebo_demos/launch/air_pressure.launch.py]: Can we keep this TODO until #78 is fixed?

File: ros_ign_gazebo_demos/launch/depth_camera.launch.py
Comment (chapulina) [inline ros_ign_gazebo_demos/launch/depth_camera.launch.py]:
```suggestion
                              description='Open RQt.'),
```

File: ros_ign_gazebo_demos/launch/imu.launch.py
Comment (chapulina) [inline ros_ign_gazebo_demos/launch/imu.launch.py:28]: This will be needed when RViz is enabled again, maybe we could leave it here commented out?

File: ros_ign_gazebo_demos/launch/rgbd_camera.launch.py
Comment (chapulina) [inline ros_ign_gazebo_demos/launch/rgbd_camera.launch.py]: Oops, there's some pre-existing issue here, the ROS 2 messages are missing the extra `msgs` namespace

**Review by chapulina — APPROVED** [review]
This is working for me, thanks! I just have some small comments below, we can merge once they're addressed.

**Pull request merged**
